src/main/cpp/auton/PrimitiveParams.h

```diff
@@ -43,7 +43,6 @@ class PrimitiveParams
         ChassisOptionEnums::HeadingOption headingOption,
         float heading,
         std::string pathName,
-        DragonCamera::PIPELINE pipelineMode,
```

Comment:
Create our own enum in PrimitiveParams that we use to create the PrimitiveParams instead of using DragonCamera's enum